src/Google.Cloud.Diagnostics.Debug/Google.Cloud.Diagnostics.Debug/LoggingClient.cs

```diff
@@ -54,10 +54,20 @@ public WriteLogEntriesResponse WriteLogEntry(StackdriverBreakpoint breakpoint)
             {
                 LogName = _logName.ToString(),
                 Severity = _logSeverityConversion[breakpoint.LogLevel],
-                TextPayload = SubstituteLogMessageFormat(
-                    breakpoint.LogMessageFormat,
-                    breakpoint.EvaluatedExpressions.ToList())
             };
+
+            if (breakpoint.Status?.IsError ?? false)
+            {
+                // The .NET Debugger does not use parameters field so we can just use format directly.
+                logEntry.TextPayload = $"LOGPOINT: Error evaluating logpoint \"{breakpoint.LogMessageFormat}\": {breakpoint.Status?.Description?.Format}.";
```

Comment:
Should 'LOGPOINT' be hardcoded here?

Reply:
Added a field with an explanation of `LOGPOINT: `